docs/source/release/v4.0.0/diffraction.rst

```diff
@@ -5,31 +5,24 @@ Diffraction Changes
 .. contents:: Table of Contents
    :local:
 
+Single Crystal Diffraction
```

Comment:
The SCD section has now got 2 `Improvements` sections that can be merged I think?